doc/update_resources.md

```diff
@@ -9,11 +9,13 @@ Check whether the patch located at `$STORM_DIR/resources/3rdparty/patches/eigen.
 
 In case a new patch needs to be created follow these steps:
 
-1. Clone `https://gitlab.com/libeigen/eigen.git` to `$STORM_DIR/resources/3rdparty/` and checkout the corresponding commit
-2. Checkout a new branch
-3. Apply the old patch via `git apply $STORM_DIR/resources/3rdparty/patches/eigen.patch`
-4. Resolve issues, make changes, and commit them
-5. Create a new patch file via `git format-patch <tag> --stdout > eigen.patch`, where `<tag>` is the tag, branch or commit from step 1
+1. Clone `https://gitlab.com/libeigen/eigen.git` somewhere and checkout the previously shipped version
+2. Checkout a new branch e.g., `git branch storm-patch; git checkout storm-patch`
+3. Apply the old patch via `git apply $STORM_DIR/resources/3rdparty/patches/eigen.patch`. At this point, `git diff` shows you all the changes we apply to Eigen
+4. Make a commit, e.g., `git commit -a -m "Storm patch"`
+5. Merge or rebase the new Eigen tag, branch or commit, e.g., `git rebase <new_commit_hash>`
+6. Resolve issues, make changes, and commit them
+5. Create a new patch file via `git format-patch <new_commit_hash> --stdout > eigen.patch`, where `<new_commit_hash>` is the tag, branch or commit from step 5
```

Comment:
```suggestion
7. Create a new patch file via `git format-patch <new_commit_hash> --stdout > eigen.patch`, where `<new_commit_hash>` is the tag, branch or commit from step 5
```

Reply:
![](https://cdn.moviepilot.de/files/e358426280019590fa2937865aab0fb5848eb897923f9a574819e6be0242/fill/2400/1152/the-count.jpg)